src/main/java/com/google/cloud/tools/maven/RunMojo.java

```diff
@@ -214,6 +214,28 @@ protected void workAroundNonJava7Version() {
     }
   }
 
+  /**
+   * Determine if the application is a Standard Environment app.
+   */
+  protected boolean isStandardEnvironmentApp() {
+    return mavenProject != null
+        && mavenProject.getBuild() != null
+        && new File(
+        mavenProject.getBuild().getDirectory() + "/" + mavenProject.getBuild().getFinalName()
```

Comment:
oh this is for the devappserver? Would it make sense to indicate the intention of the method to check the build directory to make it more explicit?